src/madlibs/MadLibs.js

```diff
@@ -0,0 +1,174 @@
+export default [
+  {
+    title: "Vacations",
+    getWord: function(key) {
+      return this.words.find((word) => {
+        return word.key === key;
+      }).value;
```

Comment:
Might be good to add a corresponding `setWord` function:

```javascript
setWord: function(key, value) {
  this.words.find(word => word.key === key).value = value;
}
```

Also, maybe too late for this project, but this _might_ be more clear as instances of a class. It would certainly be easier for us to maintain.

Reply:
Actually, never mind that won't work because it'll bypass `setState`. Maybe write this as a helper method in `App.js`?

```javascript
// App.js
updateWordWord(key, value) {
  const updatedMadLib = this.state.selectedMadLib;
  updatedMadLib.words.find(word => word.key === key).value = value;
  this.setState({selectedMadLib: updatedMadLib});
}